ui/desktop/tests/e2e/authentication.spec.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef */
+const { test } = require('./fixtures/baseTest');
+const LoginPage = require('./pages/loginPage');
+const { expect } = require('@playwright/test');
+
+/* eslint-disable no-undef */
+test.describe('user/password authentication tests', async () => {
+  test('Authenticates and signs out', async ({
+    electronPage,
+    clusterUrl,
+    username,
+    password,
+  }) => {
+    const loginPage = new LoginPage(electronPage);
+    await loginPage.setClusterUrl(clusterUrl);
+    await loginPage.loginWithPassword(username, password);
+
+    await expect(electronPage).toHaveURL(
+      /.*\/scopes\/global\/projects\/targets$/,
```

Comment:
Are we using regex instead of a string because we can ignore the base?

Reply:
Yup basically, and this assertion doesn't take glob so we have to use simple regex